### PR TITLE
#72 - Use global extension for Spock

### PR DESCRIPTION
**Pull request by phejl**
I believe this should fix https://github.com/moditect/jfrunit/issues/72. @gunnarmorling @sbrannen would you check please?

### CODE REVIEW

**Review by gunnarmorling — APPROVED** [review]
LGTM, but I'm no expert on Spock in any way. We can wait a bit to hear back from @sbrannen. Otherwise, I'm sure you've done the right thing here.

**Comment by sbrannen** [comment]
I am also not a Spock expert, but if using `IGlobalExtension` achieves the same goal then the changes look good to me.

**Comment by phejl** [comment]
> I am also not a Spock expert, but if using `IGlobalExtension` achieves the same goal then the changes look good to me.

@sbrannen Would please help me with reproducing your originally failing case in JUnit5 so I can better assure it works for you. Thanks in advance.

File: src/main/java/dev/morling/jfrunit/JfrEventTest.java [hunk 1]
```diff
@@ -23,7 +23,6 @@
 import org.junit.jupiter.api.extension.ExtendWith;
 import org.spockframework.runtime.extension.ExtensionAnnotation;
 
-@ExtensionAnnotation(JfrEventTestSpockExtension.class)
```
Comment (sbrannen) [inline src/main/java/dev/morling/jfrunit/JfrEventTest.java:26]: > @sbrannen Would please help me with reproducing your originally failing case in JUnit5 so I can better assure it works for you. Thanks in advance.

The removal of this prevents the build failure experienced in the JUnit 5 build.

**Review by sbrannen — CHANGES_REQUESTED** [review]

File: src/main/java/dev/morling/jfrunit/JfrEventTest.java [hunk 2]
```diff
@@ -23,7 +23,6 @@
 import org.junit.jupiter.api.extension.ExtendWith;
 import org.spockframework.runtime.extension.ExtensionAnnotation;
```
Comment (sbrannen) [inline src/main/java/dev/morling/jfrunit/JfrEventTest.java:24]:
```suggestion
```

Remove unused import.

**Review by sbrannen — CHANGES_REQUESTED** [review]

**Review by sbrannen — APPROVED** [review]

**Comment by gunnarmorling** [comment]
Squashed and applied. Thank you so much, @phejl and @sbrannen!

**Pull request merged**